src/data/polynomial/denoms_clearable.lean

```diff
@@ -24,7 +24,7 @@ variables {a b : R} {bi : K}
 /-- `denoms_clearable` formalizes the property that `b ^ N * f (a / b)`
 does not have denominators, if the inequality `f.nat_degree ≤ N` holds.
 
-In the implementation, we also use provide an inverse in the existential.
+In the implementation, we also provide an inverse in the existential.
```

Comment:
I still don't get what the sentence means :-)

Reply:
You are right: this is too cryptic!  What I wanted to convey is that as an output of this definition, you get `1 / i b` inside the existential quantifier.  Would what is below be clearer?

"In the implementation, we show that there exist an element `D` of `R` and an element `bi` of `K` such that clearing the denominators of the fraction equals `i D` and `bi` is an inverse of `i b`."

Reply:
In which implementation? Denominators of which fraction? I think it could still be clearer. :-)

Reply:
One of the great things about the PR process is that others have to understand what you are tying to say!  Is what's below better?  

"The definition asserts the existence of an element `D` of `R` and an element `bi = 1 / i b` of `K` such that clearing the denominators of the fraction equals `i D`."